packages/commands/src/query/chains/show.ts

```
@@ -0,0 +1,268 @@
+      "confirmations",
+      "rpc",
+      "explorer",
+    ].join(", ")}`
+  );
+};
+
+const formatChainDetails = (
+  chain: ObserverSupportedChain,
+  tokens: string[],
+  confirmations: string | undefined,
+  viemChain: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)